ingress/loadbalancer.go

```diff
@@ -2,11 +2,31 @@ package ingress
 
 import (
 	log "github.com/Sirupsen/logrus"
+
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"syscall"
+	"text/template"
 )
 
 // LoadBalancer that the controller will modify.
 type LoadBalancer interface {
-	Update([]LoadBalancerEntry) error
+	Start() error
+	Stop() error
+	Update(LoadBalancerUpdate) (bool, error)
+}
```

Comment:
Add comments for each public interface method. I thought gofmt required this actually - maybe not.